themes/default/css/_besocial/index_besocial.css

```diff
@@ -2066,6 +2066,9 @@ a.help .icon {
 		box-shadow: 3px 3px 3px rgba(0, 0, 0, 0.3);
 	}
 
+	#main_menu .linklevel1.active, #main_menu .linklevel1.active:hover {
+		border: 0px;
```

Comment:
I changed that to transparent according to Spuds suggestion: http://www.elkarte.net/community/index.php?topic=3689.msg29449#msg29449